### PR TITLE
Expanded example in crispy_tag_formsets.rst

### PR DESCRIPTION
**Pull request by bill-ash**
Hello - just a small change that caused me quite a bit of pain when reading. 

Hope it helps others!

### CODE REVIEW

File: docs/crispy_tag_formsets.rst [hunk 1]
```diff
@@ -65,8 +65,18 @@ Extra context
...
+        def __init__(self, *args, **kwargs): 
+            super(FormHelper, self).__init__(*args, **kwargs)
+            self.form_method = 'post'
+            self.layout = Layout(
+                HTML('{% if forloop.first %} Only display text on the first iteration... {% endif %}'),
+                Fieldset('Item: {{forloop.counter}}', 'field'),
+                'favorite_color', 
+                'favorite_food',
+            )
+            self.render_requried_fields = False
```
Comment (smithdc1) [inline docs/crispy_tag_formsets.rst:78]: Do we need to set this for a minimum example? 🤔

**Review by smithdc1 — COMMENTED** [review]
Thanks. Looks good, just one small question.

File: docs/crispy_tag_formsets.rst (hunk 1 above)
Comment (bill-ash) [reply docs/crispy_tag_formsets.rst:78]: Thought it was nice to display control of the formset from the helper attributes. But, not needed for a small example.

Comment (smithdc1) [reply docs/crispy_tag_formsets.rst:78]: Understood, in that case could we remove it for here. We could show `render_required_fields` but I think that should be a standalone example. 

Thanks again for the PR. 🥇

Comment (smithdc1) [inline docs/crispy_tag_formsets.rst:78]:
```suggestion
```

**Pull request merged**